tools/dotnet-test-cloud.ps1

```diff
@@ -47,17 +47,36 @@ if ($x86) {
 $testBinLog = Join-Path $ArtifactStagingFolder (Join-Path build_logs test.binlog)
 $testDiagLog = Join-Path $ArtifactStagingFolder (Join-Path test_logs diag.log)
 
-& $dotnet test $RepoRoot `
-    --no-build `
-    -c $Configuration `
-    --filter "TestCategory!=FailsInCloudTest" `
-    --collect "Code Coverage;Format=cobertura" `
-    --settings "$PSScriptRoot/test.runsettings" `
-    --blame-hang-timeout 60s `
-    --blame-crash `
-    -bl:"$testBinLog" `
-    --diag "$testDiagLog;TraceLevel=info" `
-    --logger trx `
+$mtp = Test-Path $RepoRoot\dotnet.config # poor man's test for MTP
+if ($mtp) {
+    & $dotnet test --directory $RepoRoot `
+        --no-build `
+        -c $Configuration `
+        -bl:"$testBinLog" `
+        --filter "TestCategory!=FailsInCloudTest" `
```

Comment:
I'm not sure about this. Each test framework can have their own way of filtering, so this probably needs to be revised

Reply:
Library.Template uses xunit. If people shift away from xunit, I accept that they'll have to adjust this parameter.